README.md

```diff
@@ -1,42 +1,13 @@
-Galaxy Tool Metadata Extractor
-=====================
+Galaxy Codex
+============
 
-# What is the tool doing?
+Galaxy Communities Dock aka Galaxy Codex is a catalog of Galaxy resources (tools, training, workflows) that can be filtered for any community.
 
-![plot](docs/images/Preprint_flowchart.png)
-
-
-This tool automatically collects a table of all available Galaxy tools including their metadata. Therefore, various sources are parsed to collect the metadata, such as:
-* github (parsing each tool wrapper)
-* bio.tools
-* bioconda
-* Galaxy instances (availability, statistics)
+This repository stores the sources to build this catalog. The catalog is automatically updated every week.
 
-The created table can be filtered to only show the tools relevant for a specific community. 
-
-Any Galaxy community can be added to this project and benefit from a dedicated interactive table that can be embedded into subdomains and website via an iframe. **Learn [how to add your community](https://training.galaxyproject.org/training-material//topics/dev/tutorials/community-tool-table/tutorial.html) in the dedicated GTN toturial**. 
-
-The interactive table benefits from EDAM annotations of the tools, this requires, that the tools are annotation via bio.tools.
-**Learn [how to improve metadata for Galaxy tools using the bio.tools registry](https://training.galaxyproject.org/training-material//topics/dev/tutorials/tool-annotation/tutorial.html)**.
+Any Galaxy community can be added to this project and benefit from the dedicated resources, including interactive tables that can be embedded into subdomains and website via an iframe. **Learn [how to add your community](https://training.galaxyproject.org/training-material//topics/dev/tutorials/community-tool-table/tutorial.html) in the dedicated GTN toturial**. 
 
-# Tool workflows
-
-The tool performs the following steps:
-
-- Parse tool GitHub repository from [Planemo monitor listed](https://github.com/galaxyproject/planemo-monitor)
-- Check in each repo, their `.shed.yaml` file and filter for categories, such as metagenomics 
-- Extract metadata from the `.shed.yaml`
-- Extract the requirements in the macros or xml to get version supported in Galaxy
-- Check available against conda version
-- Extract bio.tools information if available in the macros or xml
-- Check available on the 3 main galaxy instances (usegalaxy.eu, usegalaxy.org, usegalaxy.org.au)
-- Get usage statistics form usegalaxy.eu
-- Creates an interactive table for all tools: [All tools](https://galaxyproject.github.io/galaxy_tool_metadata_extractor/)
-- Creates an interactive table for all registered communities, e.g. [microGalaxy](https://galaxyproject.github.io/galaxy_tool_metadata_extractor/microgalaxy/)
-
-# Usage
-
-## Prepare environment
+# Prepare environment
```

Comment:
I would add this below `Extract all tools outside a GitHub Action`. And any objection to use conda instead of virtualenv ?

Reply:
> And any objection to use conda instead of virtualenv ?

As we could have both virtualenv and conda documented. I usually favor virtualenv when we have Python only project (even if I use it within a conda env 😅) because people might prefer avoid using conda then

Reply:
> I would add this below Extract all tools outside a GitHub Action. 

I get the point. The only thing is it is useful for Training too